src/pages/courseInstance/courseInstance.js

```diff
@@ -1,12 +1,33 @@
 const ERR = require('async-stacktrace')
 const router = require('express').Router({ mergeParams: true })
+const fileUpload = require('express-fileupload')
 const { sqlLoader } = require('@prairielearn/prairielib')
+const csvParse = require('csv-parse/lib/sync')
 const dbDriver = require('../../dbDriver')
 const asyncErrorHandler = require('../../asyncErrorHandler')
 const checks = require('../../auth/checks')
 
 const sql = sqlLoader.loadSqlEquiv(__filename)
 
+// Used to sanitize UTF-8 strings
+const utf8Re = /(?![\x00-\x7F]|[\xC0-\xDF][\x80-\xBF]|[\xE0-\xEF][\x80-\xBF]{2}|[\xF0-\xF7][\x80-\xBF]{3})./g
+
+const rosterCsvHeadingMap = {
```

Comment:
shouldn't this include all the properties on Student that we have in the DB?

Reply:
Some of them don't have whitespaces so can just nicely map it using PrairieLib.

Reply:
didn't notice that the object that was being messed with was params itself